config/300-broker.yaml

```diff
@@ -54,26 +54,15 @@ spec:
       properties:
         spec:
           properties:
-            channelTemplate:
+            channelTemplateSpec:
               type: object
               required:
-                - provisioner
+                - apiVersion
+                - kind
               properties:
-                provisioner:
-                  type: object
-                  required:
-                    - apiVersion
-                    - kind
-                    - name
-                  properties:
-                    apiVersion:
-                      type: string
-                      minLength: 1
-                    kind:
-                      type: string
-                      minLength: 1
-                    name:
-                      type: string
-                      minLength: 1
-                arguments:
```

Comment:
I see there is a RawExtension as `spec` in the `ChannelTemplateSpec`. I think arguments had that purpose in the provisioner model? Shall we rename it to spec?